src/main/resources/db/changelog/changes/v1.0.3/sql/create-list-versions-table.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS list_versions
+(
+    id                      UUID            NOT NULL PRIMARY KEY,
+    name                    VARCHAR(255)    NOT NULL,
+    description             VARCHAR(1024),
+    entity_type_id          UUID            NOT NULL,
+    fql_query               VARCHAR(1024),
+    user_friendly_query     VARCHAR(1024),
+    fields                  VARCHAR[],
+    updated_by              UUID            NOT NULL,
+    updated_by_username     VARCHAR(64)     NOT NULL,
+    updated_date            TIMESTAMP       NOT NULL,
+    is_active               boolean         NOT NULL,
+    is_private              boolean         NOT NULL,
+    version                 integer         NOT NULL,
+    list_id                 UUID            NOT NULL,
+    CONSTRAINT fk_report_id FOREIGN KEY (list_id)
+                    REFERENCES list_details (id) MATCH SIMPLE,
+);
+ CREATE INDEX IF NOT EXISTS idx_versions_list_id ON list_versions (version, list_id);
```

Comment:
Nit: I'd do `(list_id, version)` instead of `(version, list_id)`. That way, we can efficiently search for all versions for a particular list